src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessTokenClient/Altinn.Common.AccessTokenClient.csproj

```diff
@@ -13,7 +13,7 @@
   <PropertyGroup>
     <!-- NuGet package properties -->
     <PackageId>Altinn.Common.AccessTokenClient</PackageId>
-    <PackageVersion>0.5.6</PackageVersion>
+    <PackageVersion>0.5.7</PackageVersion>
```

Comment:
New nugets were just pushed for this package. Merge master and then update version, please :)

Reply:
Done